signify.c

```diff
@@ -34,6 +34,7 @@
 #include "sha2.h"
 
 #include "crypto_api.h"
+#include "base64.h"
```

Comment:
Instead of adding a new header, wouldn't it be enough to add the function prototypes to `compat.h`?